control_plane/src/remotes.rs

```diff
@@ -0,0 +1,22 @@
+use anyhow::Result;
+use std::collections::HashMap;
+use std::{fs, path::PathBuf};
+
+use crate::local_env::LocalEnv;
+
+pub type Remotes = HashMap<String, String>;
+
+fn remotes_path(local_env: &LocalEnv) -> PathBuf {
+    local_env.repo_path.join("remotes")
+}
+
+pub fn load_remotes(local_env: &LocalEnv) -> Result<Remotes> {
+    let remotes_str = fs::read_to_string(remotes_path(local_env))?;
+    Ok(toml::from_str(&remotes_str)?)
+}
+
+pub fn save_remotes(local_env: &LocalEnv, remotes: &Remotes) -> Result<()> {
+    let remotes_str = toml::to_string_pretty(remotes)?;
+    fs::write(remotes_path(local_env), remotes_str)?;
```

Comment:
This doesn't preserve any particular order in the file, I guess? Doesn't really matter, I'm just wondering..

Reply:
Yes that's correct. I think we can switch over to a BTreeMap to do that